.github/workflows/textlint.yml

```diff
@@ -0,0 +1,12 @@
+name: textlint for Japanese documentation
+on: [pull_request]
```

Comment:
Perhaps we could limit the action to just changes to `~/content/ja`?
```suggestion
on:
  pull_request:
    paths:
      - 'content/ja/**'
```

Reply:
yes think that would be better

Reply:
Thank you so much!! :laughing: